07.ls_object/ls_object.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require_relative 'entry'
+require_relative 'display'
+require 'optparse'
+
+def main
+  options = parse_options
+  entry_names = if options[:a]
+                  Dir.entries('.').sort
+                else
+                  Dir.glob('*')
+                end
+  entries = entry_names.map { |entry_name| Entry.new(entry_name) }
+  display = options[:r] ? Display.new(entries.reverse) : Display.new(entries)
+  if options[:l]
+    display.display_long
+  else
+    display.display_entries
+  end
```

Comment:
```suggestion
  display.hoge(long: options[:l])
```

Displayクラスに表示を出し分けるようにオプションで伝えるのも一手ですね

Reply:
変更しました。